app/assets/javascripts/mapknitter/Map.js

```diff
@@ -462,6 +462,28 @@ MapKnitter.Map = MapKnitter.Class.extend({
     if (this.editing._mode != "lock") e.stopPropagation()
   },
 
+  synchronizeData: function(warpables) {
+      var layers = [];
+      map.eachLayer(function(l) {layers.push(l)});
```

Comment:
can we iterate through the `featureGroup`s layers instead of the maps? So like `imgGroup.eachLayer...` or whatever its called here? Then we can ignore the image._url check I think, because if a layer is in the featureGroup it is currently placed on the map already.

Reply:
Sure, I'll have a look into it. If that's the case it would be much more tidy I think.
Thanks